### DISCUSSION TITLE
What does the abbrevatoin OD / PP mean?

### DISCUSSION THREAD

**Discussion by stefanpausch** [pmgducati/Blue-Retro-AIO-Units#34] [Q&A] [answered]
What does the abbrevatoin OD / PP mean? Thanks alot

**Comment by pmgducati** [comment, marked as answer]
OD = Open Drain
PP = Push Pull

I made both versions as Darth had them available.  According to Darth, OD is the standard that would be used with the Systems for all variants that have the option.  PP is meant for specific application (Although I am not sure that he means by that)

**Reply by stefanpausch** [reply]
Thanks a lot for the explanation. I will dig a bit for more details and add to this topic if i find more information
